providers/google_test.go

```diff
@@ -185,16 +185,33 @@ func TestGoogleProviderGetEmailAddressEmailMissing(t *testing.T) {
 
 func TestGoogleProviderUserInGroup(t *testing.T) {
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.URL.Path == "/users/member-by-email@example.com" {
-			fmt.Fprintln(w, "{}")
-		} else if r.URL.Path == "/users/non-member-by-email@example.com" {
-			fmt.Fprintln(w, "{}")
-		} else if r.URL.Path == "/users/member-by-id@example.com" {
-			fmt.Fprintln(w, "{\"id\": \"member-id\"}")
-		} else if r.URL.Path == "/users/non-member-by-id@example.com" {
-			fmt.Fprintln(w, "{\"id\": \"non-member-id\"}")
-		} else if r.URL.Path == "/groups/group@example.com/members" {
-			fmt.Fprintln(w, "{\"members\": [{\"email\": \"member-by-email@example.com\"}, {\"id\": \"member-id\", \"type\": \"USER\"}]}")
+		if r.URL.Path == "/groups/group@example.com/hasMember/member-in-domain@example.com" {
+			fmt.Fprintln(w, "{\"isMember\": true}")
+		} else if r.URL.Path == "/groups/group@example.com/hasMember/non-member-in-domain@example.com" {
+			fmt.Fprintln(w, "{\"isMember\": false}")
+		} else if r.URL.Path == "/groups/group@example.com/hasMember/member-out-of-domain@otherexample.com" {
+			http.Error(
+				w,
+				"{\"error\": {\"errors\": [{\"domain\": \"global\",\"reason\": \"invalid\",\"message\": \"Invalid Input: memberKey\"}],\"code\": 400,\"message\": \"Invalid Input: memberKey\"}",
```

Comment:
A bunch of the json strings are missing a closing `}`

Reply:
Wow, you're very right. For posterity, is there a better way of making a nice string without needing to escape every character? It looks like backticks might do it?

Reply:
Yeah, you could use backticks like this:
```
`{"error": {"errors": [{"domain": "global","reason": "invalid","message": "Invalid Input: memberKey"}],"code": 400,"message": "Invalid Input: memberKey"}`
```